docs/details/commands/pmm-admin.md

```diff
@@ -344,18 +362,43 @@ For example, if you want all collectors except `topmetrics`, specify:
 
 ###### Limit dbStats, collStats and indexStats
 
- All the collectors are enabled by default. However, collStats and indexStats are enabled if there are less than 200 collections across all the databases, excluding the admin, config, and local system databases.
+By default PMM decides what limit to set for the number of collections to monitor for the collStats and indexStats collectors. 
 
-For collStats and indexStats you can:
+If you decide to set another limit you could do that with `--max-collections-limit` parameter.
 
-- Set the value of the parameter `-max-collections-limit` to 0, which indicates that collStats and indexStats can handle unlimited collections.
- 
-Enable collStats and indexStats for some databases or collections also.
+Set the value of the parameter `--max-collections-limit` to:
+
+- 0: which indicates that collStats and indexStats can handle unlimited collections.
+- n, which indicates that collStats and indexStats can handle <=n collections. If the limit is crossed - exporter stops collecting monitoring data for the collStats and indexStats collectors.
+- -1 (default) don't need to be explicitly set, indicates that PMM decides how many collections it would monitor, currenlty <=200, but subject to change.
+
+
+To further limit collections to monitor, enable collStats and indexStats for some databases or collections:
  
-- Specify the databases and collections that collStats and indexStats will use to collect data using the parameter `--stats-collections`. This parameter receives a comma-separated list of namespaces in the form `database.collection`.
+- Specify the databases and collections that collStats and indexStats will use to collect data using the parameter `--stats-collections`. This parameter receives a comma-separated list of namespaces in the form `database[.collection]`.
+
+
+###### Examples
+
+To add MongoDB with all collectors (`diagnosticdata`, `replicasetstatus`, `collstats`, `dbstats`, `indexstats`, and `topmetrics`) with default limit detected by PMM (currently <=200 collections, but subject to change):
+
+`pmm-admin add mongodb --username=admin --password=admin_pass --enable-all-collectors mongodb_srv_1 127.0.0.1:27017`
+
+Same as above (all collectors) but increase limit:
 
+`pmm-admin add mongodb --username=admin --password=admin_pass --enable-all-collectors --max-collections-limit=1000 mongodb_srv_1 127.0.0.1:27017`
 
-###### Example
+All collectors enabled and unlimited number of collections would be monitored:
+
+`pmm-admin add mongodb --username=admin --password=admin_pass --enable-all-collectors --max-collections-limit=0 mongodb_srv_1 127.0.0.1:27017`
+
+To add MongoDB with default collectors (`diagnosticdata` and `replicasetstatus`):
+
+`pmm-admin add mongodb --username=admin --password=admin_pass mongodb_srv_1 127.0.0.1:27017`
+
+Disable `collstats` collector and enable all others without limit (`diagnosticdata`, `replicasetstatus`, `dbstats`, `indexstats`, and `topmetrics`):
```

Comment:
Disable `collstats` collector and enable all the others without limiting max-collections-limit